bokehjs/src/coffee/api/plugin.coffee

```diff
@@ -0,0 +1,27 @@
+
+define [
+  "underscore"
+  "jquery"
+  "common/logging"
+  "./figure"
+  "./helpers"
+], (_, $, Logging, figure, helpers) ->
+
+  logger = Logging.logger
+  show = helpers.show
+
+  _api = {
+    "figure": figure
+  }
+
+  $.fn.bokeh = (type, args) ->
```

Comment:
Given that this module uses bokeh's jquery, not the possibly available global one, this code isn't going to do what's expected (this will modify `Bokeh.$`).